static/js/services/delete-doc.js

```diff
@@ -8,6 +8,9 @@
     if (doc.type === 'person' && doc.parent && doc.parent._id) {
       db.getDoc(doc.parent._id, function(err, parent)  {
         if (err) {
+          if (err.reason === "deleted") {
+            return callback();
+          }
```

Comment:
That's fine. It does feel though that this should never happen. eg: if you delete a catchment area, all the contacts in that catchment should be updated (to a null catchment I guess?), in which case there should be no doc.parent._id to get. If this isn't happening then loading the Contact will show it having a Catchment Area that doesn't exist so there's still a bug to fix.